isn_tractor/ibisn.py

```diff
@@ -225,28 +222,15 @@ def map_interaction(
 
 # ## Metrics for unmapped discrete data
 
-
 def __pearson_metric(first, second):
```

Comment:
reminder for myself:

def __pearson_metric(first: t.Tensor, second: t.Tensor) -> t.Tensor:
    if first.dim() == 1 and second.dim() == 1:
        combined = t.stack([first, second], dim=1)
    else:
        combined = t.cat([first, second], axis=1) 
    return t.corrcoef(combined.T)[: first.shape[1] - 1, first.shape[1] :]